webapp/portlet/src/main/webapp/profile-contact-information/components/ProfileContactInformationDrawer.vue

```diff
@@ -215,6 +215,84 @@ export default {
   methods: {
     save() {
       this.error = null;
+      const regex =  /^[a-zA-Z0-9-_]+$/;
+      const startWithRegex =  /^\d/;
+      const isValidFirstName = regex.test(this.userToSave.firstname);
+      const isValidLastName = regex.test(this.userToSave.lastname);
+      
```

Comment:
firstname, lastname validation is based on custom validators as described in https://docs.exoplatform.org/en/5.3/Configuration.html?highlight=validator#custom-data-validators-configuration
We need to load those configuration to check each field validity.

Reply:
The constants "isValidFirstName" and "isValidLastName" aren't the final checks they are just checks for the validity of no special characters, custom validators are checked in the following. Maybe i should rename them. WDYT ?

Reply:
The regex itself could be configured by customers to enable or disable charcters, field max and min length etc ...

Reply:
Done